code/__HELPERS/roundend.dm

```diff
@@ -104,6 +104,12 @@
 	if(LAZYLEN(GLOB.round_end_notifiees))
 		send2irc("Notice", "[GLOB.round_end_notifiees.Join(", ")] the round has ended.")
 
+	if(CONFIG_GET(number/maprotation_vote_method) == MAPVOTE_VOTE && CONFIG_GET(number/maprotation_vote_delay) == -1)
```

Comment:
A vote takes 90 seconds and the server will restart at the end of the round in 60 seconds.

Did you think this through?